csp-guru/README.md

```diff
@@ -0,0 +1,50 @@
+# CSP.guru
```

Comment:
A lot of information within this README now overlaps with the top level README. That's not good, but also not super important and I also don't know of a better solution right away.

Reply:
man könnte halt nur die changes in this version da rein tun.